tests/drivers/spi/spi_loopback/boards/nucleo_f302r8.conf

```diff
@@ -1 +1,2 @@
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_2"
+CONFIG_SPI_STM32_INTERRUPT=y
```

Comment:
same here.

Reply:
no, the config SPI_STM32_INTERRUPT is set for each board upon depends on SPI